File: 2024_01/Exercicios/java/par_impar.java
import java.util.Scanner;

public class par_impar{
    public static char verificarNum(int num){
        
        if (num % 2 == 0){
            return 'P';       
        } else{
            return 'I';
        }
    }

    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        int num;

        do{
            num = scanner.nextInt();
            if(num != 0){
                System.out.println(verificarNum(num));
            }
        } while(num != 0);

        scanner.close();
    }
}

File: 2024_01/Exercicios/C/par_impar.c
#include <stdio.h>

char verificar_num(int num){

    if(num % 2 == 0){
        return 'P';;
    } else{
        return 'I';
    } 
}

int main(){
    int num = 0;

    do{
        scanf("%d", &num);
        if(num != 0){
            printf("%c\n", verificar_num(num));
        }
    } while(num != 0);

 return 0;
}